### PR TITLE
Nbo/maintenance of loading line component prod 8446

### PR DESCRIPTION
**Pull request by nborde-CSM**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -78,14 +78,15 @@
     "react-dom": "^17.0.1",
     "react-i18next": "^11.8.13",
     "react-router-dom": "^5.2.0"
-  },
+    },
```
Comment (csm-thu) [inline package.json:81]: This indent seems odd, did _prettier_ run after your modifications ?

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

File: src/misc/LoadingLine/LoadingLine.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+// Licensed under the MIT license.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Lottie from 'react-lottie-player/dist/LottiePlayerLight';
+import { useTranslation } from 'react-i18next';
+import { Grid, Typography } from '@material-ui/core';
+
+export const LoadingLine = (props) => {
+  const { t } = useTranslation();
```
Comment (jreynard-code) [inline src/misc/LoadingLine/LoadingLine.js:11]: The component should not depend on react-i18next.
Could you please remove the `useTranslation` hook usage and use props to handle title label?

File: src/misc/LoadingLine/LoadingLine.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
+---
+name: LoadingLine
+route: /LoadingLine
+menu: Misc
+---
+
+import { Playground, Props } from 'docz';
+import { LoadingLine } from './LoadingLine';
+import * as dataLoading from '../../../assets/loadingView/dataLoading.json';
```
Comment (jreynard-code) [inline src/misc/LoadingLine/LoadingLine.mdx:9]: Could you please change the folder's name `loadingView` by `loadingline` to match with the component and not the previous view anymore?

**Review by jreynard-code — CHANGES_REQUESTED** [review]
LGTM ;)
Some little remarks

**Review by csm-thu — APPROVED** [review]

**Pull request merged**